standard-cluster-services/src/main/java/org/terracotta/server/ServerEnv.java

```diff
@@ -0,0 +1,49 @@
+/*
+ *
+ *  The contents of this file are subject to the Terracotta Public License Version
+ *  2.0 (the "License"); You may not use this file except in compliance with the
+ *  License. You may obtain a copy of the License at
+ *
+ *  http://terracotta.org/legal/terracotta-public-license.
+ *
+ *  Software distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTY OF ANY KIND, either express or implied. See the License for
+ *  the specific language governing rights and limitations under the License.
+ *
+ *  The Covered Software is Entity API.
+ *
+ *  The Initial Developer of the Covered Software is
+ *  Terracotta, Inc., a Software AG company
+ *
+ */
+package org.terracotta.server;
+
+/**
+ *
+ */
+public class ServerEnv {
+
+  private static Server defaultServer;
+  private static final ThreadLocal<Server> threadServer = new ThreadLocal<>();
+
+  public static void setDefaultServer(Server server) {
+    defaultServer = server;
+  }
+
+  public static Server getDefaultServer() {
```

Comment:
Will I be able to eagerly access this method from our `ConfigurationProvider` ?

Reply:
yes